### PR TITLE
Fix : Default showArrow value

### PR DESCRIPTION
**Pull request by clemMoine**
Handle the multiple case by setting the showArrow to false by default if not set manually.

Fix : https://github.com/ant-design/ant-design/issues/12250

### CODE REVIEW

**Comment by zombieJ** [comment]
Add a snapshot test case please : )

**Comment by clemMoine** [comment]
No problem I push this now :) @zombieJ

**Comment by clemMoine** [comment]
@zombieJ I added the test case in the Select.multiple.spec.tsx

**Comment by zombieJ** [comment]
Hi, I test and found it not works well with clear icon. Css need adjust also:

**Comment by clemMoine** [comment]
@zombieJ Arf ... Yes indeed ... I can manage this but how do you want this to be handled ? I can make the clear Icon to be displayed on hover if the arrow is displayed ?

**Comment by clemMoine** [comment]
@zombieJ That is also a problem actually with the loading prop
capture d ecran 2019-02-26 a 13 09 35
, clear icon is a bit tricky to manage :D

**Comment by zombieJ** [comment]
hi @clemMoine , like single mode:

**Comment by clemMoine** [comment]
Ok ! :)

**Comment by clemMoine** [comment]
@zombieJ Are you ok if I reverse the lines `1578` and `1579` of the Select.tsx for the CSS sake :
```
1578:            {this.renderClear()}
1579:            {this.renderArrow(!!multiple)}
```
Becomes :
```
1578:            {this.renderArrow(!!multiple)}
1579:            {this.renderClear()}
```

It can allow us to have a good handling for the clear icon position handled automatically by CSS :
ezgif-5-f3f0491ba0ee
ezgif-5-3ee5c176b0f6

**Comment by zombieJ** [comment]
Keep current element order please... it may affect some user.

**Comment by clemMoine** [comment]
@zombieJ I manage the case of the multiple case with showArrow and clear icon. It remains a thing with the loader when loading is set to true that can't be handled without adding a className or something else more to indicate CSS that loader is displayed since there is no selector that allows us to select previous siblings

**Comment by clemMoine** [comment]
@zombieJ I added a className `rc-select-loading` to handle that. Please tell me if it's ok for you :)

File: src/Select.tsx [hunk 1]
```diff
@@ -1510,7 +1513,8 @@ class Select extends React.Component<Partial<ISelectProps>, ISelectState> {
       [`${prefixCls}-disabled`]: disabled,
       [`${prefixCls}-enabled`]: !disabled,
       [`${prefixCls}-allow-clear`]: !!props.allowClear,
-      [`${prefixCls}-no-arrow`]: !props.showArrow,
+      [`${prefixCls}-no-arrow`]: !showArrow,
+      [`${prefixCls}-loading`]: !!props.loading,
```
Comment (zombieJ) [inline src/Select.tsx]: Move to `const { loading, ... } = this.props;`

**Comment by zombieJ** [comment]
> @zombieJ I added a className `rc-select-loading` to handle that. Please tell me if it's ok for you :)

That's fine. Let's add this in minor version.
@chenshuai2144 , pls double confirm.

File: src/Select.tsx (hunk 1 above)
Comment (clemMoine) [reply src/Select.tsx]: Destructuring done in my last commit

**Comment by muzea** [comment]
喵喵喵？？？ 我觉得增加一个参数也行啊 见 https://github.com/react-component/select/pull/346

Why not add a parameter?  see https://github.com/react-component/select/pull/346

**Comment by zombieJ** [comment]
> Why not add a parameter? see #346

`showArrow` already can handle this. No need additional prop of `foceShowArrow` : )

**Comment by clemMoine** [comment]
@muzea in my opinion, the showArrow is already meant to handle this ;)

**Comment by muzea** [comment]
IMHO.

这里的问题是这样子的，底层组件的小版本迭代不应导致上层应用 UI 变化，我认为这算是一个 UI 的 break change。

The problem here is that the iteration of the react-component should not cause the UI to change in the upper application. I think this is a break change of the UI.

@zombieJ @clemMoine

**Comment by clemMoine** [comment]
@muzeo, considering the showArrow not to be changed in his previous default value by just adding the possibility to strictly set the showArrow even in multiple mode, I don't think this can be a breaking change but correct me if I am wrong

**Comment by zombieJ** [comment]
Hi @muzea , this will release in minor version and antd will also updated in minor version.

**Pull request merged**